src/components/authorization/roles/marketing-global.role.ts

```diff
@@ -277,7 +277,7 @@ export const Marketing = new DbRole({
       canList: true,
       properties: [
         { propertyName: 'organization', permission: { read, }, },
-        { propertyName: 'pointOfContact', permission: { read, }, },
+        { propertyName: 'pointOfContact', permission: { }, },
         { propertyName: 'types', permission: { read, }, },
         { propertyName: 'financialReportingTypes', permission: { read, }, },
         { propertyName: 'pmcEntityCode', permission: { read, }, },
```

Comment:
Similarly here what you did so far looks good, but probably remove the read permission from `pmcEntityCode` entirely, and also take the read permission for `organization` down to Low Sensitivity to match the Sheet:

![Screen Shot 2021-11-05 at 2 07 41 PM](https://user-images.githubusercontent.com/1914742/140558115-68e1194b-b4af-4a4d-9042-0e58259bf157.png)